src/main/io/vtx_string.c

```diff
@@ -24,16 +24,18 @@
 
 #include "platform.h"
 #include "build/debug.h"
+#include "common/utils.h"
 
 #if defined(VTX_COMMON)
 
-const uint16_t vtx58frequencyTable[5][8] =
```

Comment:
`6` is probably `VTX_SA_BAND_COUNT`

Reply:
this is vtx-vendor-neutral frequency table in the code shared by all vtx protocol implementations, `VTX_SA_BAND` is a Smart Audio only preprocessor define